### PR TITLE
Frame merger and Steering 

### PR DESCRIPTION
**Pull request by irapha**

### CODE REVIEW

File: buzzmobile/process/frame_merger/frame_merger.py [hunk 1]
```diff
@@ -1,6 +1,49 @@
...
 import cv2
 import numpy as np
 
+import datetime as dt
+import rospy
+from cv_bridge import CvBridge, CvBridgeError
+from sensor_msgs.msg import Image
+
+
+class Frames(object):
```
Comment (dacohen) [inline buzzmobile/process/frame_merger/frame_merger.py:11]: Can we rename this? Something like WorldModel? Not that important, but it is ambiguous.

File: buzzmobile/process/frame_merger/frame_merger.py [hunk 2]
```diff
@@ -15,31 +58,14 @@ def merge_frames(frames, weights):
...
         alpha = (weight / total_weight)
-        merged = merged + (frame * alpha).astype(np.uint8)
-
-    return merged
-
-if __name__ == '__main__':
-    np.set_printoptions(linewidth=200)
 
-    height = width = 500
+        merged += frame * alpha
```
Comment (dacohen) [inline buzzmobile/process/frame_merger/frame_merger.py:67]: Is the indentation okay here? The diff is being stupid.

**Review by dacohen — CHANGES_REQUESTED** [review]

File: buzzmobile/process/frame_merger/frame_merger.py (hunk 1 above)
Comment (irapha) [reply buzzmobile/process/frame_merger/frame_merger.py:11]: I think WorldModel is more ambiguous than frames. The way I see it, each of the parts produce a "frame", and that gets stored in Frames, which holds all frames. Then frame_merger merges those frames. I think it's more confusing if we didn't have some form of "frames" inside the "frame_merger"

File: buzzmobile/process/frame_merger/frame_merger.py (hunk 2 above)
Comment (irapha) [reply buzzmobile/process/frame_merger/frame_merger.py:67]: No this is correct

**Review by dacohen — APPROVED** [review]

**Pull request merged**